test/Listener/_files/good/merge3.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+/**
+ * @see       https://github.com/laminas/laminas-modulemanager for the canonical source repository
+ * @copyright https://github.com/laminas/laminas-modulemanager/blob/master/COPYRIGHT.md
+ * @license   https://github.com/laminas/laminas-modulemanager/blob/master/LICENSE.md New BSD License
+ */
+
+return [
+    'toUpper' => function ($input) {
+        return strtoupper($input);
+    },
+];
```

Comment:
missing empty line at the end of the file

Reply:
fixed